core/node/node_sync/src/batch_status_updater/tests.rs

```diff
@@ -159,20 +185,35 @@ fn mock_batch_details(number: u32, stage: L1BatchStage) -> api::L1BatchDetails {
             l2_tx_count: 0,
             root_hash: Some(H256::zero()),
             status: api::BlockStatus::Sealed,
-            commit_tx_hash: (stage >= L1BatchStage::Committed).then(|| H256::repeat_byte(1)),
+            commit_tx_hash: (stage >= L1BatchStage::Committed).then(|| {
+                let mut h = [0u8; 32];
+                h[0] = 1;
+                h[28..].copy_from_slice(&number.to_be_bytes());
+                H256::from(h)
+            }),
             committed_at: (stage >= L1BatchStage::Committed)
                 .then(|| Utc.timestamp_opt(100, 0).unwrap()),
+            commit_chain_id: (stage >= L1BatchStage::Committed).then_some(SLChainId(1)),
             commit_tx_finality: Some(EthTxFinalityStatus::Finalized),
-            commit_chain_id: (stage >= L1BatchStage::Committed).then_some(SLChainId(11)),
-            prove_tx_hash: (stage >= L1BatchStage::Proven).then(|| H256::repeat_byte(2)),
```

Comment:
Why, you changed it?

Reply:
I changed SLChainId to match the value returned by MockClient. I changed hash values to make them unique and easy to decode by my get_TransactionReceipt handler.